src/Microsoft.NetCore.Analyzers/UnitTests/Tasks/DoNotCallBeginInvokeOnDelegateTests.cs

```diff
@@ -0,0 +1,656 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.Testing;
+using Test.Utilities;
+using Xunit;
+
+namespace Microsoft.NetCore.Analyzers.Tasks.UnitTests
+{
+    public class DoNotCallBeginInvokeOnDelegateTests : DiagnosticAnalyzerTestBase
```

Comment:
📝 `DiagnosticAnalyzerTestBase` is the legacy test framework. New tests should be written using the new `VerifyCS` and `VerifyVB` patterns. If you don't implement this change as part of this pull request, it's fine to open a new issue to convert this.

Reply:
Resolved